docs/consensus/security.md

```diff
@@ -13,4 +13,75 @@ keywords:
+
+Unlike in proof-of-work-based blockchains, where the "challenge" for block creation comes from the previous block, in 
+Subspace this is not the case, as changing the block content does not affect the proof of space validity. Instead, 
+challenges are unique (and unpredictable), and are based on the network's proof of time component. In more detail, the 
+blockchain progress is based on "timeslots", where each timeslot is associated with a run of the proof of time 
+algorithm. We use the algorithm output to draw a block challenge for this timeslot. By design, grinding on proof of 
+time is extremely hard.
+
+For more information about the proof of time component see [this page](consensus/pot.md).
+
+## Costless simulation
+
+To mitigate against the costless simulation attack that can lead the attacker to create blocks in a greater proportion 
+than the proportion of disk space they pledge to the network, we use correlated randomness in block challenges.
+
+Subspace uses the c-correlation method, where challenges for 'c' blocks are correlated and deterministic. Using this 
+approach, an attacker who tries to simulate many potential forks gains significantly less power, as the ability to 
+manoeuvre across these forks becomes more and more limited as 'c' gets larger.
+
+### C-correlation predictability window
+
+Even though the challenges for 'c' blocks are deterministic, since we use proof of time to draw the challenges, they are 
+not known in advanced, but only revealed when the timeslot arrives (by definition). This prevents potential issues, like 
+so-called "bribing attacks", that come with the correlation of block challenges and the predictability window associated 
+with c-correlation in general. 
+
+## Long-range attacks
+
+To prevent long-range attacks, we use proof of time as a fundamental component in our consensus protocol.
+
+Attackers that try to bootstrap a competing and longer (i.e. heavier) chain cannot do it without cost, since they must 
+show that sufficient time has passed for the lifespan of this fork. In other words, like in proof of work, they must 
+spend a significant amount of sequential work in maintaining the attack.
+
+For more information about how proof of time is used see [this page](consensus/pot.md).
+
+# Security against attacks on proof of capacity / storage
+
+The masking function that we apply during farmer's plot creation has specific properties that help us in preventing the 
+following attacks on the protocol.
+
+## Time-Memory algorithms (plot compression)
+
+We adopt the function described in the work ["Beyond Hellman's Time-Memory Trade-Offs with Applications to Proofs of Space"](https://eprint.iacr.org/2017/893) 
+as our masking function. This function is designed such that the gain in trading computation (time) over storage (memory) 
+is very small.
+
+## On-the-fly plot creation
+
+Preventing farmers from creating plots on the fly, after seeing the challenge, has two favours.
```

Comment:
Did you mean "flavors" here?

Reply:
Indeed, thanks. I guess it should all be in US English spelling?

Reply:
Probably should be US English for consistency. This one I was more pointing out it was the wrong word.